src/AzureAuth/Program.cs

```diff
@@ -97,6 +97,12 @@ private static void Main(string[] args)
             services.AddSingleton<IPublicClientAuth, PublicClientAuth>();
             services.AddSingleton<ILogger>(logger);
 
+            Console.CancelKeyPress += (_, _) =>
+            {
+                logger.LogWarning("Received SIGINT (Ctrl+C), exiting...");
+                Environment.Exit(-1);

```

Comment:
I'm not familiar with returning a negative exit code. Is that a thing? I would have expected to see `1` rather than `-1`.

Reply:
After doing a brief search around I don't think any systems expect a negative exit code and this would be similar to returning 256, so I think we want `1` here.

Reply:
I thought -1 was like 255, which was the exit code when I interrupt a program in some debuggers.😅

Reply:
If you want 255, you should use 255, not -1, which would rely on consumers to wrap around.

From https://unix.stackexchange.com/questions/110348/how-do-i-get-the-list-of-exit-codes-and-or-return-codes-and-meaning-for-a-comm, 

it looks like in Bash at least, generally `130` indicates interrupted.

Reply:
Seems like I should use exit code 2. 130=128+2 where 128 means the process was force closed by external signals on Linux.

Reply:
For example, in this post: https://stackoverflow.com/questions/66321338/exit-code-130-on-linux-and-2-on-windows-on-sigint